src/js/common/schemaHolders.js

```diff
@@ -0,0 +1,183 @@
+            properties: {
+                // TODO: Revert
+                // password: "{that}.options.schema.definitions.password",
+                // confirm: "{that}.options.schema.definitions.confirm",
+                password: "{that}.options.definitions.password",
+                confirm: "{that}.options.definitions.confirm",
+                code: {
+                    type: "string",
+                    required: true,
+                    minLength: 1,
+                    errors: {
+                        "": "gpii.express.user.reset.password.mismatch"
+                    }
+                }
+            }
+        }
+    });
+
+    // The schema holder for the initial user signup form.
+    fluid.defaults("gpii.express.user.schemaHolder.signup", {
+        gradeNames: ["gpii.express.user.schemaHolder"],
+        schema: {
+            title: "gpii-express-user user signup schema",
+            description: "This schema defines the format accepted when creating a new user.",
+            properties: {
+                // TODO: Revert
+                // email: "{that}.options.schema.definitions.email",
+                // username: "{that}.options.schema.definitions.username",
+                // password: "{that}.options.schema.definitions.password",
+                // confirm: "{that}.options.schema.definitions.confirm",
+                // profile: "{that}.options.schema.definitions.profile"
+                email: "{that}.options.definitions.email",
+                username: "{that}.options.definitions.username",
+                password: "{that}.options.definitions.password",
+                confirm: "{that}.options.definitions.confirm",
+                profile: "{that}.options.definitions.profile"
+            }
+        }
+    });
+
+    // The schema holder requesting that the initial verification code be resent.
+    fluid.defaults("gpii.express.user.schemaHolder.resend", {
+        gradeNames: ["gpii.express.user.schemaHolder"],
+        schema: {
+            title: "gpii-express-user user resend verification code schema",
+            description: "This schema defines the format accepted when requesting that a verification code be resent.",
+            properties: {
+                // TODO: Revert
+                // email: "{that}.options.schema.definitions.email"
+                email: "{that}.options.definitions.email"
+            }
+        }
+    });
+
+    // The schema holder for the user login endpoint.
+    fluid.defaults("gpii.express.user.schemaHolder.login", {
+        gradeNames: ["gpii.express.user.schemaHolder"],
+        schema: {
+            title: "gpii-express-user login schema",
+            description: "This schema defines the format accepted when logging in.",
+            properties: {
+                username: {
+                    anyOf: [
+                        // TODO: Revert
```

Comment:
I think this TODO can be done now?

Reply:
Thanks for catching this, fixing it now.